### ISSUE TITLE
[dev.icinga.com #10649] Host and service group actions

### ISSUE THREAD

**Issue by icinga-migration** [Icinga/icingaweb2#2140]
This issue has been migrated from Redmine: https://dev.icinga.com/issues/10649

**Created by smtalk on 2015-11-16 15:44:01 +00:00**

Assignee: _elippmann_
Status: _Assigned_ 
Target Version: _Backlog_
Last Update: _2016-01-25 12:37:10 +00:00 (in Redmine)_

---

I haven't seen a feature request for that. It'd be nice if servicegroups / hostgroups would have action buttons in Icinga Web 2. At least for: enable/disable notifications for all services/hosts, schedule downtime. Icinga Web 1 had the actions available in /cgi-bin/extinfo.cgi?type=5&hostgroup=hotgroupname.

---

**Relations**:

* relates [#10649](https://dev.icinga.com/issues/10649)
* duplicates [#10649](https://dev.icinga.com/issues/10649)

**Comment by shakalandy** [comment]
Ah, thanks for already opening an issue for that. 
Was just wondering if i'm really the only person in need for this.
Would be great if this could be implemented.
relates to #2726

**Comment by andral** [comment]
Would be a nice feature to have!

**Comment by ktella** [comment]
Can we bump up the priority of this ? I really need this in our environment.

**Comment by lippserd** [comment]
Sure, if you want to participate in sponsoring this feature feel free to [contact](https://www.icinga.com/contact/) us.

**Comment by zmaupin-do** [comment]
Any update on this? Seems a pretty big feature to be lacking.

**Comment by facetoe** [comment]
+1 for this feature.

**Comment by mickenordin** [comment]
Yes we'd like this too, if at least there was a "select all" button for the views, everything would be much simpler.
